metadata.rb

```diff
@@ -1,9 +1,9 @@
-name             "php-ioncube"
-maintainer       "Andriy Samilyak"
-maintainer_email "samilyak@gmail.com"
-license          "All rights reserved"
-description      "Installs/Configures php-ioncube"
+name             'php-ioncube'
+maintainer       'Andriy Samilyak'
+maintainer_email 'samilyak@gmail.com'
+license          'All rights reserved'
+description      'Installs/Configures php-ioncube'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          "0.2.0"
+version          '0.2.2'
```

Comment:
Because there already is '0.2.1' version: https://supermarket.chef.io/cookbooks/php-ioncube